### PR TITLE
Tweaks and additions to the 0.0.27 pre-release notes.

### PR DESCRIPTION
**Pull request by creachadair**
Add mention of Objective C and TypeScript. Fix various cosmetic capitalization
and punctuation and formatting bits.

### CODE REVIEW

**Comment by creachadair** [comment]
Do we want to mention also some of the Bazel changes, e.g., that nearly-all the dependencies are now configured as external repositories? This doesn't affect users of the release tarball, but is a useful point of interest for someone checking out at this tag.

File: RELEASES.md [hunk 1]
```diff
@@ -4,68 +4,72 @@
 
 Due to the period of time between this release and v0.0.26, many relevant
 changes and fixes may not appear in the following list.  For a complete list of
-changes, please check: https://github.com/google/kythe/commits/
+changes, please check the commit logs:
+https://github.com/google/kythe/search?q=committer-date%3A2016-11-01..2018-05-31&type=Commits
```
Comment (creachadair) [inline RELEASES.md]: We will need to update the dates here, once we pick a release date. :)

Comment (schroederc) [reply RELEASES.md]: `https://github.com/google/kythe/compare/v0.0.26...v0.0.27`

File: RELEASES.md [hunk 2]
```diff
@@ -4,68 +4,72 @@
...
 
-Notable changes:
- - remove GRPC server and client code
+### Notable additions
```
Comment (schroederc) [inline RELEASES.md]: Please don't change the format yet.  It doesn't match at all with the rest of the log.  We're just pushing out a release to catch-up.

File: RELEASES.md [hunk 3]
```diff
@@ -4,68 +4,72 @@
...
+
+ - First release of Go indexer and Go extractors.
+ - Objective C is now supported in the C++ indexer.
+ - First release of TypeScript indexer.
```
Comment (schroederc) [inline RELEASES.md]: This is not included in the release.

Comment (creachadair) [reply RELEASES.md]: Should it be?

File: RELEASES.md (hunk 1 above)
Comment (creachadair) [reply RELEASES.md]: Oh, nice, thanks. Done.

File: RELEASES.md (hunk 2 above)
Comment (creachadair) [reply RELEASES.md]: Reverted the header tags.

File: RELEASES.md (hunk 3 above)
Comment (schroederc) [reply RELEASES.md]: I was pushing it the next release to simplify.

Comment (creachadair) [reply RELEASES.md]: Makes sense—I removed it.

**Pull request merged**